lib/msal-browser/src/interaction_client/SilentIframeClient.ts

```diff
@@ -137,12 +137,13 @@ export class SilentIframeClient extends StandardInteractionClient {
                 this.logger,
                 this.performanceClient,
                 request.correlationId
-            )(
+            )({
                 serverTelemetryManager,
-                silentRequest.authority,
-                silentRequest.azureCloudOptions,
-                silentRequest.account
-            );
+                requestAuthority: silentRequest.authority,
+                requestAzureCloudOptions: silentRequest.azureCloudOptions,
+                requestExtraQueryParameters: silentRequest.extraQueryParameters,
```

Comment:
Can we avoid passing `extraQueryParams` everywhere, if the `instance_aware` param is added at `RequestParameterBuilder`?

Reply:
We need to pass it to the authority resolver to replace the hostname, if needed. That's why there are so many changes across the module.